react/features/analytics/middleware.js

```diff
@@ -80,6 +83,31 @@ function calculateLocalTrackDuration(state) {
  */
 MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
+    case LIB_WILL_INIT: {
+        const { getState } = store;
+        const state = getState();
+        const { analytics } = state['features/base/config'];
+
+        if (analytics.rtcstatsEnabled) {
```

Comment:
Just a suggestion: analytics middleware handles generic analytics, but this code is RTC stats specific. I would at least put it into separate method and call here.

Reply:
Makes sense, would It be ok If I moved to it's own feature?